damus/Views/QRCodeView.swift

```diff
@@ -159,7 +150,10 @@ struct QRCodeView: View {
 
             Spacer()
             
-            Text("Follow me on Nostr", comment: "Text on QR code view to prompt viewer looking at screen to follow the user.")
+            
+            Text(verbatim: "Follow " +
+                            ((damus_state.pubkey.npub == pubkey.npub) ? "me" : "\(profile?.display_name ?? "")") +
+                             " on Nostr")
```

Comment:
Use verbatim only on strings that shouldn't be translated. It's also better to separate this into two conditional strings since one doesn't need to be parameterized ("me"). It's harder or sometimes impossible for translators to translate properly depending on the language and phrasing.

```suggestion
            if damus_state.pubkey.npub == pubkey.npub {
                Text("Follow me on Nostr", comment: "Text on QR code view to prompt viewer looking at screen to follow the user.")
            } else {
                Text("Follow \(profile?.display_name ?? profile?.name ?? "") on Nostr", comment: "Text on QR code view to prompt viewer looking at screen to follow the user.")
            }
```

Reply:
Thanks @tyiu, I've updated the code based on your suggestion